modules/swagger-codegen/src/main/java/io/swagger/codegen/CliOption.java

```diff
@@ -20,4 +33,49 @@ public String getDescription() {
     public void setDescription(String description) {
         this.description = description;
     }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public String getDefault() {
+        return defaultValue;
+    }
```

Comment:
When looking at the `java` client library, I see no `default` values.  Please make sure this field is properly populated and tested.